conda/recipes/libwholegraph/meta.yaml

```diff
@@ -55,6 +55,7 @@ requirements:
     - cudatoolkit
     {% else %}
     - cuda-cudart-dev
+    - cuda-nvml-dev
```

Comment:
This change needed an entry in `dependencies.yaml` as well. See https://github.com/rapidsai/devcontainers/pull/366#issuecomment-2243859440. In devcontainers, we see:

```
2024-07-22T21:24:02.4790079Z CMake Error at CMakeLists.txt:204 (target_link_libraries):
2024-07-22T21:24:02.4793870Z   Target "wholegraph" links to:
2024-07-22T21:24:02.4798184Z 
2024-07-22T21:24:02.4802582Z     CUDA::nvml
2024-07-22T21:24:02.4804498Z 
2024-07-22T21:24:02.4804898Z   but the target was not found.  Possible reasons include:
2024-07-22T21:24:02.4805252Z 
2024-07-22T21:24:02.4805598Z     * There is a typo in the target name.
2024-07-22T21:24:02.4810296Z     * A find_package call is missing for an IMPORTED target.
2024-07-22T21:24:02.4811022Z     * An ALIAS target is missing.
```

Reply:
I filed https://github.com/rapidsai/wholegraph/pull/197 to fix.

Reply:
Good catch. Thanks Bradley! 🙏